### DISCUSSION TITLE
Updating a section heading.

### DISCUSSION THREAD

**Discussion by IRodriguezSC** [sadieblue/agilityblue.workspace-scripts#29] [Q&A] [answered]
Does anyone has an example of updating a section heading?

**Comment by bmward80** [comment]
Are you asking about the section heading of the form within a task, or just a normal form?

Brandon

**Comment by IRodriguezSC** [comment]
So, for a task with a form, I’m looking to update one (or more) form section heading(s).
I’ve tried a few things but no luck yet.
-IR

Ismael Rodriguez | Mgr - Elec Discov/Lit Support
Sullivan & Cromwell LLP
125 Broad Street | New York, NY 10004-2498
+1 212 558 7245 (T)
***@***.******@***.***> | www.sullcrom.com<http://www.sullcrom.com/>

From: Brandon Ward ***@***.***>
Sent: Monday, September 30, 2024 7:17 PM
To: sadieblue/agilityblue.workspace-scripts ***@***.***>
Cc: Rodriguez, Ismael ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [sadieblue/agilityblue.workspace-scripts] Updating a section heading. (Discussion #29)


Are you asking about the section heading of the form within a task, or just a normal form?

Brandon

—
Reply to this email directly, view it on GitHub [github.com]<https://urldefense.com/v3/__https:/github.com/sadieblue/agilityblue.workspace-scripts/discussions/29*discussioncomment-10803129__;Iw!!KhposeJF9Q!RNqzhGmYrwmNi3I4UKGqkYCUeIQzbW2O80R0yRISFOhaQ_sShfr-8J7wdqZQbnVj_lWJQP764Zw_29atBd9mD1IabXg$>, or unsubscribe [github.com]<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/BK4IHLDVAV3BBQGMFSHCCMLZZHLVHAVCNFSM6AAAAABPEG7UP6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAOBQGMYTEOI__;!!KhposeJF9Q!RNqzhGmYrwmNi3I4UKGqkYCUeIQzbW2O80R0yRISFOhaQ_sShfr-8J7wdqZQbnVj_lWJQP764Zw_29atBd9mAJ1cjhA$>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

________________________________
**This is an external message from: ***@***.******@***.***> **

________________________________
This e-mail is sent by a law firm and contains information that may be privileged and confidential. If you are not the intended recipient, please delete the e-mail and notify us immediately.

**Reply by IRodriguezSC** [reply]
So, for a task with a form, I’m looking to update one (or more) form section heading(s).  
I’ve tried a few things but no luck yet.

**Comment by bmward80** [comment, marked as answer]
The properties for task form sections are read-only. They are derived from the form that they were created from and cannot be edited. See the Task Form Section Entity Model table [here on the api docs](https://apidocs.agilityblue.com/#7188f9cc-69d1-4f96-818f-bb8b2d08e739).

Brandon

**Reply by IRodriguezSC** [reply]
Thank you Brandon.  This makes sense.
